cmd/aws-vpc-cni/main.go

```diff
@@ -384,14 +387,17 @@ func validateEnvVars() bool {
 	}
 
 	prefixDelegationEn := utils.GetBoolAsStringEnvVar(envEnPrefixDelegation, defaultEnPrefixDelegation)
-	warmIPTarget := utils.GetEnv(envWarmIPTarget, "0")
-	warmPrefixTarget := utils.GetEnv(envWarmPrefixTarget, "0")
-	minimumIPTarget := utils.GetEnv(envMinIPTarget, "0")
+	warmIPTarget := utils.GetEnv(envWarmIPTarget, defaultWarmIPTarget)
+	warmPrefixTarget := utils.GetEnv(envWarmPrefixTarget, defaultWarmPrefixTarget)
+	minimumIPTarget := utils.GetEnv(envMinIPTarget, defaultMinIPTarget)
```

Comment:
With the integer values, we can use `utils.GetIntFromStringEnvVar`

Reply:
updated